group/views.py

```diff
@@ -182,3 +182,26 @@ def post(self, request):
                             status=status.HTTP_400_BAD_REQUEST)
         return Response({"error": "User is not a member of this group"},
                         status=status.HTTP_403_FORBIDDEN)
+
+    def get(self, request, user_id):
```

Comment:
Suggest adding a default on the `user_id` here if you really want to use the same class for both URLs